api/document/js/akn/main.ts

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+import * as CodeMirror from 'codemirror';
+import 'codemirror/mode/xml/xml.js';
+import 'codemirror/addon/search/search.js';
+import 'codemirror/addon/search/searchcursor.js';
+import 'codemirror/addon/dialog/dialog.js';
+
+import { getEl, getElAttr } from '../util';
+
+
+// We need to load our CSS via require() rather than import;
+// using the latter raises errors about not being able to find
+// the module.
+
+require('codemirror/lib/codemirror.css');
+require('codemirror/theme/eclipse.css');
+require('codemirror/addon/dialog/dialog.css');
```

Comment:
For a follow up PR: let's add these to our compiled style sheets; better to be consistent than use CSS-in-JS some of the time and straight SASS compilation others.